### ISSUE TITLE
Its not obvious that shelr recording something

### ISSUE THREAD

**Issue by antono** [shelr/shelr#2]
we should change PS1 to something like PS1="[recording] $PS1"

**Comment by torrottum** [comment]
Or could be to add a red asterisk or circle, to represent a recording circle. (since [recording] is a little long, in my opinion)

**Comment by antono** [comment]
yep, something like red circle will work.

**Comment by torrottum** [comment]
I think doing this would be sufficient enough:
export PS1="[\033[31m] ● [\033[00m]$PS1"

**Comment by samtsai** [comment]
I'm glad to see this issue here, because I was about to open one for a "recording" status. I like toringerottum's implementation.

**Comment by antono** [comment]
Yep, it should be added. However it will not work with custom PS1 generators like those provided by bash-it or oh-my-zsh. Going to add it though.
